### PR TITLE
Update dev figure hashes

### PR DESCRIPTION
**Pull request by nabobalis**
Need to see if this breaks something.

### CODE REVIEW

File: sunpy/map/tests/test_reproject_to.py [hunk 1]
```diff
@@ -51,6 +51,7 @@ def test_reproject_to_hgs(aia171_test_map, hgs_header):
     aia171_test_map.reproject_to(hgs_header).plot()
 
 
+@figure_test
```
Comment (nabobalis) [inline sunpy/map/tests/test_reproject_to.py:54]: I get that these are not standard figure tests but I still think they should be marked as such.

File: sunpy/image/tests/test_transform.py [hunk 2]
```diff
@@ -296,8 +296,8 @@ def test_clipping(rot30):
     return fig
 
 
-@pytest.mark.filterwarnings("ignore:.*bug in the implementation of scikit-image")
 @figure_test
+@pytest.mark.filterwarnings("ignore:.*bug in the implementation of scikit-image")
```
Comment (ayshih) [inline sunpy/image/tests/test_transform.py:300]: Revert this because it makes no difference

File: sunpy/map/tests/test_mapbase.py [hunk 3]
```diff
@@ -1654,8 +1654,8 @@ def test_rotation_rect_pixelated_data(aia171_test_map):
     rect_rot_map.peek()
 
 
-@pytest.mark.parametrize('method', _rotation_registry.keys())
 @figure_test
+@pytest.mark.parametrize('method', _rotation_registry.keys())
```
Comment (ayshih) [inline sunpy/map/tests/test_mapbase.py:1658]: Revert this because it makes no difference

**Review by ayshih — CHANGES_REQUESTED** [review]
I'm certain that swapping the order of the decorators makes no difference for the issue that was encountered in CI, so I'd like those swaps to be reverted to keep the Git history more clean.

File: sunpy/map/tests/test_mapbase.py (hunk 3 above)
Comment (nabobalis) [inline sunpy/map/tests/test_mapbase.py:1658]:
```suggestion
@pytest.mark.parametrize('method', _rotation_registry.keys())
@figure_test
```

File: sunpy/image/tests/test_transform.py (hunk 2 above)
Comment (nabobalis) [inline sunpy/image/tests/test_transform.py:300]:
```suggestion
@pytest.mark.filterwarnings("ignore:.*bug in the implementation of scikit-image")
@figure_test
```

**Review by ayshih — APPROVED** [review]

**Pull request merged**